docs/autorenew-feature.md

```diff
@@ -0,0 +1,82 @@
+# The Entity Auto Renewal feature
+
+## Goal
+- Prior to the implementation of the Entity Auto Renewal feature, the expiration time of a Hedera entity, which is specified in the HAPI document, has not been checked or enforced. An entity remains active in the ledger even after its expiration time, without additional fees being charged. Upon implementation of this feature, Hedera Services will __begin to charge rent__ for automatically renewed entities; and will remove from the ledger expired entities which are either deleted or have an admin/autorenew account with zero balance at the time renewal fees are due.
+
+## Design
+- Introduce new settings in `application.properties`:
+  * `autorenew.isEnabled`
+  * `autorenew.numberOfEntitiesToScan`
+  * `autorenew.maxNumberOfEntitiesToRenewOrDelete`
+  * `autorenew.gracePeriod`
```

Comment:
Will these be exposed in some network query, given it changes.
The gracePeriod information might be useful information for exchanges to provide customers with.

Reply:
They will all be global dynamic properties, which means:
- They will be part of file `0.0.121` (APP_PROPERTIES)
- They can be exposed via a get file content transaction of `0.0.121`
- They can be updated via a file update transaction to `0.0.121`
- The changes will take effect right after (hence dynamic) the file update

Reply:
Great, thanks